### PR TITLE
Add support for integer axis, axis labels and title for `plot_interactive`

### PR DESCRIPTION
**Pull request by mkaze**
This PR addresses issues #125 and #127 for Altair plots, i.e. `plot_interactive`. The changes are all backwards compatible (including the axes labels, text of title as well as the logic for removing axes vectors; although, for the last one it might have side effects, e.g. an embedding with name "Dimension 0" would be removed if axis is set to 0).

@koaning It seems the `plot_interactive` is not tested at all in the test suit. So I just complied and didn't add any tests and instead manually tested the changes to make sure they are working. Do you have any specific plan for adding the tests for this method?

### CODE REVIEW

**Comment by koaning** [comment]
@mkaze this is certainly on the mental todo list but it feels best to add this as an issue so that we might discuss what features we'd like to test. Altair offers a relatively convenient API to test the properties of a chart.

**Comment by mkaze** [comment]
@koaning Do you possibly have any idea why the tests failed? It seems it's an style check issue, however because of using `--exit-zero` flag all of them should be treated as just warning?! And the `black` does not raise any error locally! So I couldn't figure out what the problem is?!

**Comment by koaning** [comment]
My guesstimate is that we're not specifying the target version. Looking at the [docs](https://github.com/psf/black) it seems that you can set what version of python you'd like to refer to as a benchmark. Quote; 

```
  -t, --target-version [py27|py33|py34|py35|py36|py37|py38]
                                  Python versions that should be supported by
                                  Black's output. [default: per-file auto-
                                  detection]
```

This might explain part of the behaviour. If you're running python 3.7 locally then the black run for python 3.6 might have a different opinion. I doubt this explains everything, but it's worth to add.

**Comment by koaning** [comment]
It also seems like `black` recently had an [update](https://pypi.org/project/black/#history). It might be that some breaking changes got introduced. We currently run this line in the CI: 

```
pip install flake8 purest black
```

Which means that we're always using the latest version. We might want to consider not doing that anymore.

**Comment by koaning** [comment]
Considering that a docs update is also failing https://github.com/RasaHQ/whatlies/pull/176/checks?check_run_id=1032428463 I'm inclined to think it is caused by the black update.

**Comment by mkaze** [comment]
@koaning That's right! I can confirm that the `black` maintainers has released a new version with more opinionated formatting and therefore this breaks the style checks. However, personally I am not in favor of pinning `black` version because it's not a dependency per say, and also we want our code to be compatible and consistent with the latest black updates (probably that's why we are using it in the first place!).

**Comment by mkaze** [comment]
@koaning I suggest to create a new PR addressing the latest changes suggested by `black` (and/or `flake8`), and then after merging that PR we can rebase our current PRs based on it. I probably can do that if you are busy now :)

**Comment by koaning** [comment]
I was trying to do that right now actually, but there's something strange happening. 

It seems that the version of black that you have when you run `black .` is different than the one that the `pre-commit` hook fetches. It's a bummer because the changes that the black cli wants to introduce are now made undone immediately by the pre-commit hook. 

Have you had any luck?

**Comment by mkaze** [comment]
@koaning Run `pre-commit autoupdate` to update the hooks to the latest version, then perform the commit.

**Review by koaning — APPROVED** [review]

File: whatlies/embeddingset.py
Comment (koaning) [inline whatlies/embeddingset.py:896]: Don't we want to add a default value for these? Unless specified I can imagine a use-case to have `x_axis`=0 and `y_axis=1`. That way you don't have to fiddle around when you're using `Umap` or `Pca`.

Comment (mkaze) [reply whatlies/embeddingset.py:896]: @koaning Actually, I had that in mind and wanted to create an issue for it since this is also a valid point for `plot` method as well (and even `Embedding.plot`). If you are OK with it, I'll address this in a separate PR with `x_axis=0` and `y_axis=1` as the default for all plot methods. Is that fine with you?

Comment (koaning) [reply whatlies/embeddingset.py:896]: Ah yeah, that's a better idea. Makes the testing a lot clearer too.

**Comment by koaning** [comment]
LGTM!

**Pull request merged**
